internal/webapi/server.go

```diff
@@ -18,12 +18,30 @@ type api struct {
 	db database.Database
 }
 
-func NewServer(db database.Database) *Server {
+type APIAuthPacket struct {
```

Comment:
Why's this called a Packet? I think `AuthCredentials` makes more sense. Packet is something that exists in TCP/IP, but not up here.